apps/designer/app/routes/$.tsx

```diff
@@ -17,11 +16,28 @@ import {
   getBuildParams,
   dashboardPath,
 } from "~/shared/router-utils";
+import { db } from "@webstudio-is/project/server";
+import type { DynamicLinksFunction } from "remix-utils";
 
 type Data =
   | (CanvasData & { env: Env; mode: BuildMode })
   | { errors: string; env: Env };
 
+export const dynamicLinks: DynamicLinksFunction<CanvasData> = ({
+  data,
+  location,
+}) => {
+  const searchParams = new URLSearchParams(location.search);
+  searchParams.set("pageId", data.page.id);
+  return [
+    {
+      rel: "stylesheet",
+      href: `/s/css/?${searchParams}`,
```

Comment:
This is probably not a correct way to use URLSearchParams. The resulting URL looks like this: "/s/css/?mode=preview**&amp;**pageId=c27ea236-b510-46cf-92a9-c127816f948e"

Reply:
checking

Reply:
URLSearchParams doesn't encode & with `&amp;` it seems to happen from something else and it seems to be working, so I guess its fine? 


it happens somewhere between remix and node I guess

Reply:
Seems like React does this and this is correct https://stackoverflow.com/a/3705601/478603